framework/packages/abstract-sdk/src/cw_helpers/cw_storage_plus.rs

```diff
@@ -1,81 +0,0 @@
-use cosmwasm_std::{StdResult, Storage};
-use cw_storage_plus::{Map, PrimaryKey};
-use serde::{de::DeserializeOwned, Serialize};
-
-/// Load a batch of values by their keys from a [`Map`].
-pub fn load_many<'a, K, V>(
-    map: Map<K, V>,
-    storage: &dyn Storage,
-    keys: Vec<K>,
-) -> StdResult<Vec<(K, V)>>
-where
-    K: PrimaryKey<'a>,
-    V: DeserializeOwned + Serialize,
-{
-    let mut res: Vec<(K, V)> = vec![];
-
-    for key in keys.into_iter() {
-        let value = map.load(storage, key.clone())?;
-        res.push((key, value));
-    }
-
-    Ok(res)
-}
```

Comment:
This helper was causing ton of redundant clones in queries. Code is much cleaner without this helper IMO